_pages/courses.md

```diff
@@ -10,46 +10,49 @@ permalink: /courses
 
 # Game-Theoretic Approach to Planning and Synthesis
 
-- **Instructors**: [Giuseppe De Giacomo](https://www.dis.uniroma1.it/degiacom/), [Antonio Di Stasio](https://www.dis.uniroma1.it/users/antonio_di-stasio), [Giuseppe Perelli](https://giuseppeperelli.github.io/), [Shufang Zhu](https://shufang-zhu.github.io/) 
-- **Type**: PhD Course
-- **Number of hours**: 20
-- **Class delivery mode**: Blended
-- **Dates**: From Monday 4th July to Friday 8th July from 14:00 to 18:00 CEST
-- **In presence location**: Room A7, Department of Computer, Automation, and Management Engineering, Sapienza University of Rome
+<p align="center" style="font-size:200%" > 4-8 July 2022 </p>
+
+- **Instructors**: [Giuseppe De Giacomo](https://www.dis.uniroma1.it/degiacom/), [Antonio Di Stasio](https://www.dis.uniroma1.it/users/antonio_di-stasio), [Giuseppe Perelli](https://giuseppeperelli.github.io/), [Shufang Zhu](https://shufang-zhu.github.io/) (Sapienza University of Rome)
+- **Host Institutions**: Sapienza University & ICT-48 TAILOR
+- **Level**: Postgraduate.
+- **Course duration**: 20 hours.
+- **Course Type**: Lecture series.
+- **Registration**: Free but required (Please register here: [https://forms.gle/G6jj71kE92ZDqWU29](https://forms.gle/G6jj71kE92ZDqWU29))
+- **Class delivery modality**: Blended (Online and In Presence).
+- **In presence location**: Room 7, Department of Computer, Automation, and Business Engineering, Sapienza University of Rome
```

Comment:
```suggestion
- **In presence location**: Room A7, Department of Computer, Automation, and Management Engineering, Sapienza University of Rome
```

Reply:
Don't know if you wanted to change it or not. In case, this is my suggestion as last PR :)